dashboard/app/models/user.rb

```diff
@@ -858,8 +858,8 @@ def teacher?
   end
 
   def authorized_teacher?
-    # you are "really" a teacher if you are a teacher in any cohort for an ops workshop or in a plc course
-    admin? || (teacher? && (cohorts.present? || plc_enrollments.present?)) ||
+    # you are "really" a teacher if you have the AUTHORIZED_TEACHER user permissions, or are a teacher in any cohort for an ops workshop or in a plc course
+    permission?(UserPermission::AUTHORIZED_TEACHER) || admin? || (teacher? && (cohorts.present? || plc_enrollments.present?)) ||
       permission?(UserPermission::LEVELBUILDER)
```

Comment:
Feel welcome to disagree and keep as-is, if you feel the existing is more readable.